components/molecules/charts/schedule/helper/columns.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Column
+ */
+
+// React
+import React from 'react'
+import styled from 'styled-components'
+import {
+  endOfMonth,
+  endOfDay,
+  endOfWeek,
+  startOfMonth,
+  startOfWeek,
+  startOfDay,
+  setMonth,
+  setDay
+} from 'date-fns'
+// IUI
+import ColumnPagination from './columnPagination'
+import Tooltip from '../../../../atoms/tooltip/tooltip'
+
+const DATE_TYPE = {
+  DAY: 'day',
+  MONTH: 'month',
+  YEAR: 'year',
+  WEEK: 'week'
+}
+
+const columnPattern = {
+  day: [...Array(12).keys()],
+  month: ['Week1', 'Week2', 'Week3', 'Week4'],
+  year: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
+  week: ['Sat', 'Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri']
+}
+
+const formatCell = (handleClick, month, row, otherData) => {
+  return row[month] && row[month][0] ? (
+    row[month][2] ? (
+      <Tooltip content={row[month][2]}>
+        <Wrapper
+          context={row[month][1] ? row[month][1] : 'light'}
+          count={row[month][0] || row[month]}
+          onClick={(e) =>
+            (row[month][0] > 0 || row[month] > 0) && handleClick({ e, month, row }, otherData)
+          }
+        >
+          {row[month][0] ? row[month][0] : row[month]}
+        </Wrapper>
+      </Tooltip>
+    ) : (
+      <Wrapper
+        context={row[month][1] ? row[month][1] : 'light'}
+        count={row[month][0] || row[month]}
+        onClick={(e) =>
+          (row[month][0] > 0 || row[month] > 0) && handleClick({ e, month, row }, otherData)
+        }
+      >
+        {row[month][0] ? row[month][0] : row[month]}
+      </Wrapper>
+    )
+  ) : (
+    '0'
+  )
+}
+
+export default (handleClick, options) => {
+  const {
+    mode = DATE_TYPE.MONTH,
+    setCurrentDate,
+    currentDate,
+    hiddenColumn,
+    currentDataSource,
+    onTitleFormatter
+  } = options
+  if (!currentDataSource.length) return []
```

Comment:
@Ojaghi1991 Always use brackets with If () { } and add space above for readabillity